xds/internal/balancer/cdsbalancer/cdsbalancer_security_test.go

```diff
@@ -224,11 +225,14 @@ func makeNewSubConn(ctx context.Context, edsCC balancer.ClientConn, parentCC *te
 // the address attributes added as part of the intercepted NewSubConn() method
 // indicate the use of fallback credentials.
 func (s) TestSecurityConfigWithoutXDSCreds(t *testing.T) {
+	oldOutlierDetection := envconfig.XDSOutlierDetection
```

Comment:
Can this be moved to the setup functions? Like `setupWithWatch`?

Reply:
And why do you need to make the new env var true for all the tests?
I was expecting one set of new tests that set this to true, but the others are untouched.

Reply:
Yeah, fair enough. I added it to everything to have the no-op Outlier Detection LB config be generated. I'll leave the env var off for all these tests, and then send nil LB Config.